code/src/App.js

```diff
@@ -1,9 +1,28 @@
 import React from 'react'
+import { Provider } from 'react-redux'
+import { configureStore, combineReducers } from '@reduxjs/toolkit'
+
+import { remember } from 'reducers/remember'
+import { Summary } from 'components/Summary'
+import { List } from 'components/List'
+import { AddTask } from 'components/AddTask'
+import { AppShell } from 'lib/Container'
+
+const reducer = combineReducers({
+  remember: remember.reducer
+})
+
+const store = configureStore({ reducer })
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <Provider store={store}>
+      <AppShell>
+        <Summary />
+        <AddTask />
+        <List />
+        {/* <Footer /> */}
```

Comment:
Footer not added but I guess that is on your todo-list :)

Reply:
If not, let's remove it ;)